### ISSUE TITLE
Make all pages have the same layout

### ISSUE THREAD

**Issue by jelevoir** [bpowell/cse480#118]
All pages need to have a consistent layout.

**Comment by bpowell** [comment]
:+1:

**Comment by jelevoir** [comment]
@bpowell, @dcfreema, @pforgaciu, @ehill2

This is an ongoing issue.
**If you add a page, please add it in a comment here.**

I will be keeping track of which pages have been altered, and which pages still have to have to be.

**Comment by jelevoir** [comment]
Pages that have been completed:
- admin/addbar
- admin/addbusiness
- admin/admin
- admin/changeroles
- barview
- changepassword
- display
- drinklist
- error/404
- login
- owner/addbartender
- owner/owner
- owner/removebartender
- profile
- register
- welcome

Pages yet to be completed:
NONE

**Comment by bpowell** [comment]
owner/removebartender once #138 is merged.

**Comment by jelevoir** [comment]
Change all pages to have the following non-mobile column sizes:
- col-md-2 | ---------col-md-8--------- | col-md-2

instead of:
- col-md-3 ---| ------col-md-6------ |--- col-md-3

This is to help fill up more of the screen. To make it not look so empty on a desktop.
The center column is the actual content where the outside columns are empty.

Pages that have been completed:
- profile - #94 

Pages yet to be completed:
- admin/addbar
- admin/addbusiness
- admin/admin
- admin/changeroles
- barview
- changepassword
- display
- drinklist
- error/404
- login
- owner/addbartender
- owner/owner
- owner/removebartender
- register
- welcome

**Comment by bpowell** [comment]
Closing this issue. For any new pages a new issue will be created.
